doc/api/customization.md

```diff
@@ -0,0 +1,51 @@
+# Customization
+
+> Stability: 1.0 - Early development
+
+Node.js has multiple APIs for customizing behaviour and can easily be invoked
```

Comment:
```suggestion
Node.js has multiple APIs for customizing behavior and can easily be invoked
```

Reply:
Gah. I think VS Code uses my system language for spell check. (This project has confused the heck outta spell check on iOS)

Reply:
I use this extension: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

It allows you to have custom settings by workspace, so you can create a Node workspace and configure `cSpell` within that workspace to use US English.